chain/jsonrpc/src/lib.rs

```diff
@@ -632,11 +630,7 @@ impl JsonRpcHandler {
                 tx_info,
```

Comment:
Maybe it would be nice to log the error here as part of the tracing warn since it is being silently converted to a TimeoutError